include/eve/module/core/regular/impl/simd/x86/minmax.hpp

```diff
@@ -0,0 +1,43 @@
+//==================================================================================================
+/*
+  EVE - Expressive Vector Engine
+  Copyright : EVE Project Contributors
+  SPDX-License-Identifier: BSL-1.0
+*/
+//==================================================================================================
+#pragma once
+
+#include <eve/concept/value.hpp>
+#include <eve/detail/apply_over.hpp>
+#include <eve/detail/category.hpp>
+#include <eve/detail/implementation.hpp>
+
+namespace eve::detail
+{
+// -----  Regular
+template<arithmetic_scalar_value T, typename N>
+EVE_FORCEINLINE auto
+minmax_(EVE_SUPPORTS(sse2_), wide<T, N> v0, wide<T, N> v1) noexcept requires x86_abi<abi_t<T, N>>
+{
+  using enum category;
+  constexpr auto c = categorize<wide<T,N>>();
+
+  if constexpr(match(c, integer_)  && current_api == avx && N::value > 1)
```

Comment:
This seems bad since the sse api will be targeted by this.

Reply:
How is SSE taking this as I asked for api to be == avx ?

Reply:
Api is avx. Abi is sse

Reply:
Indeed